packages/snaps-controllers/src/services/webview/WebViewMessageStream.test.ts

```diff
@@ -0,0 +1,73 @@
+import { sleep } from '@metamask/snaps-utils/test-utils';
+
+import { WebViewMessageStream } from './WebViewMessageStream';
+
+describe('WebViewMessageStream', () => {
+  let mockWebView: any;
+  let getWebView: any;
+  let stream: WebViewMessageStream;
+
+  beforeEach(() => {
+    mockWebView = {
```

Comment:
Could we create a utility function for creating and returning the objects required for the test?

I don't love using beforeEach and the describe scope for storing references to the objects we are testing on.

Reply:
Got u. Created utility function and removed before each them. Organised tests so get values in the first use case.